src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -720,7 +722,7 @@ public async Task GetSemanticTokens_Razor_CoalesceDeleteAndAddAsync()
 
             var (previousResultId, service, _, _) = await AssertSemanticTokensAsync(new string[] { txt, newTxt }, new bool[] { false, true });
 
-            var (newResultId, _, _) = await AssertSemanticTokenEditsAsync(newTxt, expectDelta: true, isRazor: true, previousResultId: previousResultId, service: service);
+            var (newResultId, _, _) = await AssertSemanticTokenEditsAsync(newTxt, expectDelta: true, isRazor: false, previousResultId: previousResultId, service: service);
```

Comment:
I _ think_ this should be false, since the initial call to `AssertSemanticTokens` acts on a `.cshtml` file instead of a `.razor` file. Feel free to correct me if I'm wrong, cc @ryanbrandenburg

Reply:
Hmm, did this fail without the change? I believe that you're correct but it's weird that it didn't fail before this.

Reply:
The test starts failing with this change since we now cache tokens by document URI. Since the docs have different file extensions, they're considered different URIs and thus the cached tokens aren't being found. Before since we cached purely based on resultId, it was a non-issue.